XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -53,13 +51,13 @@ public class HTTP {
                         
                     case let s where s.rangeOfString("application/json") != nil:
                         
-                        let (json: AnyObject!, error) = JSON.parse(data)
-                        let headers = httpResponse.allHeaderFields
+                        let (json, error) = JSON.parse(data!)
```

Comment:
That's because we're only checking against `nil` but not doing optional unwrapping... Will change it!